airbyte-webapp/src/pages/BuilderPage/components/YamlEditor.module.scss

```diff
@@ -0,0 +1,33 @@
+@use "../../../scss/colors";
+
+.container {
+  display: flex;
+  flex-flow: column;
+  height: 100%;
+}
+
+.control {
+  flex: 0 0 auto;
+  background-color: colors.$dark-blue;
+  display: flex;
+  padding: 10px;
+}
+
+.editorContainer {
+  flex: 1 1 0;
+}
+
+.downloadButton {
+  margin-left: auto;
+}
+
+// Export colors to be used in monaco editor
+:export {
+  // Monaco editor requires 6-character hex values for theme colors
+  /* stylelint-disable-next-line color-hex-length, color-no-hex */
+  tokenString: #ffffff;
```

Comment:
Yeah I tried using that, but because it was a 3-character hex code instead of a 6-character one, monaco editor threw an error saying it did not recognize the color. And I couldn't change the `_colors.scss` value to a 6-value one or else stylelint threw a `color-hex-length` error, so thats why I added that here with the comment and disabled that stylelint rule.

But maybe it would be better to just change the `_colors.scss` value to a 6-character one and disable the stylelint rule there?

Reply:
I ended up changing the `_colors.scss` value to a 6-character hex value and disabling the color-hex-length rule there as well, so that the `$white` value will work with monaco. See the changes [here](https://github.com/airbytehq/airbyte/pull/17482/files/ceb79ba1d6de1ced58a46dd91fa41f7638011613..2bd22556722cf94d16947087cc4427d0b8c0bbb8)